docs/workflows/SNVPhyl/1.1/snvphyl-workflow-1.1.ga

```diff
@@ -0,0 +1 @@
ts/filterstat/1.8.1", "type": "tool"}, "16": {"tool_id": "localhost:9009/repos/nml/snv_matrix/snvmatrix/1.8.1", "tool_version": "1.8.1", "outputs": [{"type": "tabular", "name": "out"}], "workflow_outputs": [{"output_name": "out", "uuid": "09b50681-fdc4-4c17-95dc-4c2264e70f56", "label": null}], "input_connections": {"phylip": {"output_name": "out_file1", "id": 14}}, "tool_state": "{\"__page__\": 0, \"phylip\": \"{\\\"__class__\\\": \\\"ConnectedValue\\\"}\", \"__rerun_remap_job_id__\": null}", "id": 16, "tool_shed_repository": {"owner": "nml", "changeset_revision": "41c192a4dba3", "name": "snv_matrix", "tool_shed": "localhost:9009"}, "uuid": "19028cf9-fd41-470b-8cc7-f987db36fcfe", "errors": null, "name": "SNV Matrix", "post_job_actions": {"RenameDatasetActionout": {"output_name": "out", "action_type": "RenameDatasetAction", "action_arguments": {"newname": "snvMatrix.tsv"}}}, "label": null, "inputs": [], "position": {"top": 366, "left": 3134}, "annotation": "", "content_id": "localhost:9009/repos/nml/snv_matrix/snvmatrix/1.8.1", "type": "tool"}, "17": {"tool_id": "localhost:9009/repos/iuc/phyml/phyml/3.3.20190321", "tool_version": "3.3.20190321", "outputs": [{"type": "nhx", "name": "output_tree"}, {"type": "txt", "name": "output_stats"}, {"type": "txt", "name": "output_stdout"}], "workflow_outputs": [{"output_name": "output_stats", "uuid": "e2309c74-728c-42cf-a649-c692502ba20e", "label": null}, {"output_name": "output_tree", "uuid": "dcfdad80-62a3-4789-892b-a51912d7af85", "label": null}, {"output_name": "output_stdout", "uuid": "70ec58ac-b524-4578-a6a0-f0234725dda4", "label": null}], "input_connections": {"input": {"output_name": "out_file1", "id": 14}}, "tool_state": "{\"equi_freq\": \"\\\"m\\\"\", \"__page__\": null, \"input\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\", \"seq\": \"{\\\"__current_case__\\\": 0, \\\"model\\\": \\\"GTR\\\", \\\"tstv\\\": \\\"e\\\", \\\"type_of_seq\\\": \\\"nt\\\"}\", \"__rerun_remap_job_id__\": null, \"move\": \"\\\"NNI\\\"\", \"support_condition\": \"{\\\"__current_case__\\\": 3, \\\"branchSupport\\\": \\\"-4\\\"}\", \"numStartSeed\": \"\\\"0\\\"\", \"prop_invar\": \"\\\"e\\\"\", \"optimisationTopology\": \"\\\"tlr\\\"\", \"gamma\": \"\\\"e\\\"\", \"phylip_format\": \"\\\"\\\"\", \"nb_data_set\": \"\\\"1\\\"\", \"userInputTree\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\", \"nbSubstCat\": \"\\\"4\\\"\"}", "id": 17, "tool_shed_repository": {"owner": "iuc", "changeset_revision": "98914d6c5a6c", "name": "phyml", "tool_shed": "localhost:9009"}, "uuid": "e81872c9-28ec-42f5-8605-785c40eb7ee0", "errors": null, "name": "PhyML", "post_job_actions": {}, "label": null, "inputs": [{"name": "userInputTree", "description": "runtime parameter for tool PhyML"}, {"name": "input", "description": "runtime parameter for tool PhyML"}], "position": {"top": 149, "left": 3210}, "annotation": "", "content_id": "localhost:9009/repos/iuc/phyml/phyml/3.3.20190321", "type": "tool"}}, "annotation": "snvphyl", "a_galaxy_workflow": "true"}
```

Comment:
Could you change the formatting of this workflow to multi-line?

Also, it looks like this workflow makes use of the wrong version of smalt (`1.0.0` instead of `1.1.0`).

Reply:
Smalt 1.0.0 is the correct version since made a new wrapper that contains the smalt index built in.